arkade/src/commonTest/kotlin/com/arkade/core/contracts/ArkContractTest.kt

```
@@ -0,0 +1,8 @@
+package com.arkade.core.contracts
+
+import kotlin.test.Test
+
+class ArkContractTest : com.arkade.Test() {
+    @Test
+    fun should_create_contract_successfully() {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)